server/paymentProviders/stripe/webhook.ts

```diff
@@ -280,6 +286,8 @@ export const paymentIntentProcessing = async (event: Stripe.Event) => {
       },
       { transaction },
     );
+
+    sendEmailNotifications(order);
```

Comment:
What is this about?

Reply:
The processing webhook was not sending a notification or creating an activity for the processing status.